indico/modules/events/management/util.py

```diff
@@ -182,3 +183,38 @@ def _process_attachments(folders):
 
     else:
         raise TypeError('Unexpected object of type {}: {}'.format(type(root).__name__, root))
+
+
+def find_unregistered_users(person_list):
+    """Get a set of users which don't have an Indico account
+
+        :param person_list: List of persons to check
+
+    """
+    return set(filter(lambda p: p.person.user is None, person_list))
```

Comment:
Well I use it in 4 other places and in each you pass different list so in each it would have to be different, I think it makes sense to have a function and I think its also easier to understand what it does by looking at function name rather then just looking at the code